homeassistant/components/dsmr/strings.json

```diff
@@ -76,6 +76,12 @@
       "gas_meter_reading": {
         "name": "Gas consumption"
       },
+      "current_average_demand": {
+        "name": "Current average demand"
+      },
+      "maximum_demand_month": {
+        "name": "Maximum demand current month"
+      },
```

Comment:
Would it make sense to include `Power` in the entity name? And could we maybe remove current from that one name? Could maybe be confusing with the actual current

Reply:
In the official docs its called:
`Current average demand - Active energy import` and
`Maximum demand – Active energy import of the running month`

In dutch it's just called 'Piekverbruik'

How would you name them? Cause there is also a (not implemented) value with the last peak usage for the last 13 months.
So thats why you need something to state it's for the current month :)

Reply:
Average energy demand
Maximum energy demand current month

wdyt

Reply:
oh and is it power or energy?

Reply:
It's a 15min average for the power usage. 

`Average power demand`
`Max average power demand current month`

Somethime like this?

Reply:
Let's follow the official fluvius documentation and use:
current_average_demand en maximum_demand_current_month
In the future we can then also add maximum_demand_last_13_months